adapter/src/test/java/com/twilio/kudu/sql/ScenarioIT.java

```diff
@@ -48,15 +51,17 @@ public class ScenarioIT {
   @ClassRule
   public static KuduTestHarness testHarness = new KuduTestHarness();
   private static String JDBC_URL;
+  private static String JDBC_URL_VIEWS_DISABLED;
   private static Scenario usageReportTransactionScenario;
 
   @BeforeClass
   public static void setup() throws SQLException, IOException {
     initializeHints();
     String urlFormat = JDBCUtil.CALCITE_MODEL_TEMPLATE_DML_DDL_ENABLED + ";schema."
-        + KuduSchema.CREATE_DUMMY_PARTITION_FLAG + "=false";
+        + KuduSchema.CREATE_DUMMY_PARTITION_FLAG + "=false" + ";schema." + KuduSchema.DISABLE_SCHEMA_CACHE + "=true";
     JDBC_URL = String.format(urlFormat, DefaultKuduSchemaFactory.class.getName(),
         testHarness.getMasterAddressesAsString());
+    JDBC_URL_VIEWS_DISABLED = JDBC_URL + ";schema." + KuduSchema.DISABLE_MATERIALIZED_VIEWS + "=true";
```

Comment:
why are we disabling materialized views here?

Reply:
That is so that we can run a query that uses the fact table and compare running the same query against the view.